pkg/rancher-desktop/utils/string-encode.ts

```diff
@@ -0,0 +1,11 @@
+export const hexEncode = (str: string): string => {
```

Comment:
Hmm, isn't an arrow function _very slightly_ more expensive than a plain function?

Reply:
Not that I'm aware of and a quick search shows that the answer is no, not in any meaningful way. 

That said, I'm open to changing. If we don't like the style or there are concerns about performance with arrow functions we should probably update our linter to discourage their usage.